.circleci/config.yml

```diff
@@ -275,6 +276,19 @@ jobs:
             NODE_ENV=production npm run sdk -- notifications --output-dir=$CIRCLE_ARTIFACTS/notifications-panel
       - store-artifacts-and-test-results
 
+  build-wpcom-block-editor:
+    <<: *defaults
+    parallelism: 1
+    steps:
+      - prepare
+      - run:
+          name: Build the block editor in WordPress.com integration utils package
+          command: |
+            npx lerna run clean --scope='@automattic/wpcom-block-editor'
+            npx lerna run bundle --scope='@automattic/wpcom-block-editor' -- -- --output-path=$CIRCLE_ARTIFACTS/wpcom-block-editor
```

Comment:
I was a bit surprised to see `bundle` and `build` here:

https://github.com/Automattic/wp-calypso/blob/33454d6e370e4df2935c4e21cbdb5d74a5340fbe/apps/wpcom-block-editor/package.json#L18-L23

I suspect that's more scripts than we need, but we can clean up as we iterate on `calypso-build`.